biggus/__init__.py

```diff
@@ -485,6 +485,21 @@ class Array(object):
 
     __hash__ = None
 
+    def __str__(self):
+        nbytes = self.nbytes
+        if nbytes < (1 << 10):
+            size = '{} B'.format(nbytes)
+        elif nbytes < (1 << 20):
+            size = '{:.02f} KiB'.format(nbytes / (1 << 10))
```

Comment:
They're not quite the same. `nbytes / 1024` gives a floating point result, `nbytes >> 10` gives an integer (equivalent to `nbytes // 1024`).